packages/commonwealth/server/routes/communities/create_community_handler.ts

```diff
@@ -14,9 +17,21 @@ export const createCommunityHandler = async (
   req: TypedRequestBody<CreateCommunityRequestBody>,
   res: TypedResponse<CreateCommunityResponse>,
 ) => {
+  for (const key in req.body) {
+    if (req.body[key] === '' || req.body[key] === null) {
+      delete req.body[key];
+    }
+  }
+
+  const validationResult = await createCommunitySchema.safeParseAsync(req.body);
+
+  if (validationResult.success === false) {
+    throw new AppError(formatErrorPretty(validationResult));
+  }
+
```

Comment:
This is a step in the right direction - clear message validation section 💯 
From here we can further refactor into a more declarative validation middleware, once we start establishing a model driven structure - nice work.